src/AudioToolbox/MusicTrack.cs

```diff
@@ -129,6 +129,20 @@ internal override IntPtr ToUnmanaged ()
 
 	public class MusicEventUserData : MidiRawData {
 		public MusicEventUserData () {}
+
+		internal MusicEventUserData (IntPtr handle)
```

Comment:
Might be a good idea to check `handle` for IntPtr.Zero before trying to read from it (and throw an ArgumentNullException).